layers/+emacs/outshine/README.org

```diff
@@ -0,0 +1,45 @@
+#+TITLE: outshine layer
+
+# TOC links should be GitHub style anchors.
+* Table of Contents                                        :TOC_4_gh:noexport:
+- [[#description][Description]]
+  - [[#features][Features:]]
+- [[#install][Install]]
+- [[#key-bindings][Key bindings]]
+
+* Description
+This layer adds support for [[https://github.com/alphapapa/outshine][outshine]] and [[https://github.com/alphapapa/outorg][outorg]]. They will be enabled in all
+programming modes.
+
+** Features:
+- Navigate through code buffers via headings like you do with org buffers
+- Edit comments under outline headings in separate org-mode buffers
+
+* Install
+To use this configuration layer, add it to your =~/.spacemacs=. You will need to
+add =outshine= to the existing =dotspacemacs-configuration-layers= list in this
+file.
+
+* Key bindings
+
+| Key Binding     | Description                                                |
+|-----------------+------------------------------------------------------------|
+| ~SPC a o o S~   | Show the contents of all outline sections                  |
+| ~SPC a o o g u~ | Navigate up an outline heading                             |
+| ~SPC a o o g n~ | Navigate to the next outline heading                       |
+| ~SPC a o o g j~ | Navigate to the next outline heading at the same level     |
+| ~SPC a o o g j~ | Navigate to the previous outline heading at the same level |
+| ~SPC a o o i h~ | Insert an outline heading                                  |
+| ~SPC a o o J~   | Move the current outline subtree down                      |
+| ~SPC a o o K~   | Move the current outline subtree up                        |
+| ~SPC a o o >~   | Demote the current outline subtree                         |
+| ~SPC a o o <~   | Promote the current outline subtree                        |
+| ~SPC a o O e~   | Edit the current heading as org                            |
+| ~SPC m O q~     | Save the edits back to the original file                   |
```

Comment:
First of all thanks for the PR, I am already using it.

I think the command to save the edits back should be `SPC m c` to be consistent with other layers.

Reply:
Ah, I fixed this in my commits but need to update the documentation.